frontend/src/app/features/intercambios/pages/intercambio-page/intercambio-page.component.ts

```diff
@@ -0,0 +1,131 @@
+import { PerfilUsuario } from './../../../../../generated/openapi/model/perfilUsuario'
+import { Component, OnInit } from '@angular/core'
+import { ActivatedRoute } from '@angular/router'
+import { ServicioUsuario } from 'src/app/core/services/servicio-usuario.service'
+import {
+  CreacionIntercambio,
+  DefaultService,
+  DetallesLibro,
+  Dueno,
+  LibroConDuenos,
+  ModelError,
+  ValidaPuedeIntercambiar200Response,
+} from 'src/generated/openapi'
+
+@Component({
+  selector: 'app-intercambio-page',
+  templateUrl: './intercambio-page.component.html',
+  styleUrls: ['./intercambio-page.component.sass'],
+})
+export class IntercambioPageComponent implements OnInit {
```

Comment:
```suggestion
export class LibroPageComponent implements OnInit {
```

Más que mostrar intercambios, esta página muestra la información de un libro, por lo que le cambiaría el nombre